README.md

```diff
@@ -1,6 +1,51 @@
 # Northcoders News API
 
-Please create enviroment variables to be able to clone and locally run this file succesfully. Please create the following files:
+This is an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture. It accesses different databases (articles, comments, topics, and users) and uses these to create multiple end points each with a specific use, detailed in get/API.
+
+_Inserted hosted link_
+
+Please start by cloning this repo.
+
+- Fork repo, and copy code URL.
+- In VS code run command 'git clone url'
+
+# ----Set up infomation----
+
+- Run NPM install. This will download dependencies from package.JSON.
+
+-Run NPM setup-dbs. This will initialise the data base ready to be populated
+
+-Run NPM seed. This will seed the newly set up database with values and data.
+
+-Run NPM test. This is will enable testing suite to run.
```

Comment:
I recommend adding backticks around commands to make the markdown render nicely, e.g. \`npm install\` will look like `npm install`